### PR TITLE
fdctl: better workspace ENOMEM diagnostics

### PR DESCRIPTION
**Pull request by mmcgee-jump**

### CODE REVIEW

File: src/app/fdctl/configure/hugetlbfs.c [hunk 1]
```diff
@@ -109,6 +110,56 @@ init( config_t * const config ) {
   }
 }
 
+static void
+cmdline( char * buf,
+         ulong  buf_len ) {
```
Comment (ptaffet-jump) [inline src/app/fdctl/configure/hugetlbfs.c:115]: `len` typically means strlen. This should be `buf_sz`

File: src/app/fdctl/configure/workspace.c [hunk 2]
```diff
@@ -14,7 +14,12 @@
 static void
 init_perm( fd_caps_ctx_t *  caps,
            config_t * const config ) {
-  ulong mlock_limit = fd_topo_mlock( &config->topo );
+  ulong mlock_limit = 0UL;
```
Comment (ptaffet-jump) [inline src/app/fdctl/configure/workspace.c:17]: Does this need to be a sum instead of a max if running in single process mode?

**Review by ptaffet-jump — DISMISSED** [review]
Great! Some nits, but it's fine to merge if you'd like

File: src/app/fdctl/configure/workspace.c (hunk 2 above)
Comment (mmcgee-jump) [reply src/app/fdctl/configure/workspace.c:17]: Nope, these workspaces are created and then unmapped one by one.

**Review by ptaffet-jump — APPROVED** [review]

**Pull request merged**